### ISSUE TITLE
宠物置顶

### ISSUE THREAD

**Issue by Yuxun0225** [ChaozhongLiu/DyberPet#22]
虽然也许是我自己的问题，但实在不知道怎么解决，其他时候可以正常的在最上层，但是在玩原神的时候点到宠物就会跳回桌面

**Comment by ChaozhongLiu** [comment]
确实可能是原神这一个App全屏时自身的问题
关于置顶 我没有做过多设置 基本就是PySide6的内置行为
我这两天测试测试 看看怎么解决 谢谢！

**Comment by LingYiWu015** [comment]
emm,hei the developer,for your kindness,we will have such a great frame to build my own subpet on the computer


but , to be honest,I was confused that have this bug fixed?
I've checked the developing logs(which you write it in Chinese.)
I didn't see any words about this kind of bugs were fixed
Could you tell me have you fixed it?
(or I will try to found out what's up about the logical problems.)
basically, I just what to ensure the bug's situation.
thanks to your work before again.

there are the pics i found in your *readme.md* file.
![This is what I see](https://github.com/user-attachments/assets/398e8c25-b817-4ff3-9e08-d778a38b4613)

the only thing you mentioned it is in your update at the **Version 0.4.0**(But I still don't know that have you fixed it)

**Comment by ChaozhongLiu** [comment]
你好 我一直没是时间解决这个问题，还没有探究过到底是系统原因、原神原因还是软件的原因。如果你有意向研究这个问题，欢迎在解决之后直接 PR。谢谢你！

**Comment by Yuxun0225** [comment]
我後來發現，只要在視窗的部分弄能無邊框視窗就沒有這個問題

ChaozhongLiu ***@***.***> 於 2024年7月24日 週三 上午2:35 寫道：

**Comment by ChaozhongLiu** [comment]
感谢你的反馈！很有用！
